lighthouse-core/test/lib/manifest-parser-test.js

```diff
@@ -27,52 +28,70 @@ function noUrlManifestParser(manifestSrc) {
 describe('Manifest Parser', function() {
   it('should not parse empty string input', function() {
     const parsedManifest = noUrlManifestParser('');
-    assert.ok(parsedManifest.warning);
+    expect(parsedManifest.warning)
+      .toEqual('ERROR: file isn\'t valid JSON: SyntaxError: Unexpected end of JSON input');
   });
 
   it('accepts empty dictionary', function() {
     const parsedManifest = noUrlManifestParser('{}');
-    assert(!parsedManifest.warning);
-    assert.equal(parsedManifest.value.name.value, undefined);
-    assert.equal(parsedManifest.value.short_name.value, undefined);
-    assert.equal(parsedManifest.value.start_url.value, EXAMPLE_DOC_URL);
-    assert.equal(parsedManifest.value.display.value, 'browser');
-    assert.equal(parsedManifest.value.orientation.value, undefined);
-    assert.equal(parsedManifest.value.theme_color.value, undefined);
-    assert.equal(parsedManifest.value.background_color.value, undefined);
-    assert.ok(Array.isArray(parsedManifest.value.icons.value));
-    assert.ok(parsedManifest.value.icons.value.length === 0);
+    expect(parsedManifest.warning).toBeUndefined();
+    expect(parsedManifest.value.name.value).toBe(undefined);
+    expect(parsedManifest.value.short_name.value).toBe(undefined);
+    expect(parsedManifest.value.start_url.value).toBe(EXAMPLE_DOC_URL);
+    expect(parsedManifest.value.display.value).toBe('browser');
+    expect(parsedManifest.value.orientation.value).toBe(undefined);
+    expect(parsedManifest.value.theme_color.value).toBe(undefined);
+    expect(parsedManifest.value.background_color.value).toBe(undefined);
+    expect(parsedManifest.value.icons.value).toHaveLength(0);
     // TODO:
     // related_applications
     // prefer_related_applications
   });
 
+  it('should warn on invalid manifest parser URL', function() {
+    const parsedManifest = manifestParser('{}', 'not a URL', EXAMPLE_DOC_URL);
+    expect(parsedManifest.warning)
+      .toEqual('ERROR: invalid manifest URL: \'not a URL\'');
+  });
+
+  it('should warn on valid but non-HTTPS manifest parser URL', function() {
```

Comment:
nit:
```suggestion
  it('should warn on valid but non-(HTTP|HTTPS) manifest parser URL', function() {
```
or whatever :)